docs/user-guide/features/multi-tenancy-setup.md

```diff
@@ -0,0 +1,75 @@
+# Multi-Tenancy Setup Guide
+
+This doc describes how to setup multi-tenancy on a testing cluster.
+
+## Tenant Bootstrap
+Tenant can be created using the following command:
+
+```bash
+kubectl create tenant $[tenant name]
+```
+
+When a new tenant is created, a cluster role and rolebinding is automatically created for this tenant for user "admin". This admin cluster role allows full access to this tenant's space resource management.
```

Comment:
rephrase: to resource management in this tenant's space.

Reply:
fixed